docs/style-guide.rst

```diff
@@ -1049,9 +1049,11 @@ Layout contract elements in the following order:
 
 1. Pragma statements
 2. Import statements
-3. Interfaces
-4. Libraries
-5. Contracts
+3. Events
+4. Errors
+5. Interfaces
+6. Libraries
+7. Contracts
```

Comment:
This should be non-controversial. They obviously should not be above imports or below contracts. And the "events before errors" order is already used below in the recommendation for contracts (though personally I'd put them in the opposite order - errors seem more general).

I intentionally did not add structs, enums and UDVTs - not sure where they'd go in that order (obviously also between imports and contracts, but where releative to errors and events).

Reply:
Could you update line 1048 (can't leave a comment there directly) from:
```
Layout contract elements in the following order:
```
to 
```
Lay out contract elements in the following order:
```
or even better
```
Contract elements should have the following layout order:
```
since layout is not a verb.

Reply:
Fixed.